src/vmeasure.jl

```diff
@@ -0,0 +1,70 @@
+"""V-measure of contingency table"""
+function _vmeasure(A::Matrix{Int}; β = 1.0)
+    C, K = size(A)
+    N = sum(A)
+
+    # Homogeneity
+    hck = 0.0
+    for k in 1:K
+        d = sum(A[:,k])
+        for c in 1:C
+            if A[c,k] != 0 && d != 0
+                hck += log(A[c,k]/d) * A[c,k]/N
```

Comment:
Is it possible to use `entropy()` from `StatsBase.jl`, since we are already requiring this package?
Also scaling by `d` and `N` could be done once for the whole column.

Reply:
Because `d` value depends on a particular column, it would require to run loop over column (row) to calculate such scaling factor, ∑Ak/N log(N/∑Ac).